src/components/SubjectList/Goods/GoodList.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import Goods from './Goods';
+import './Good.scss';
+
+class GoodList extends Component {
+  render() {
+    const { goods } = this.props;
+    return (
+      <>
+        {goods.map(item => (
+          <Goods
+            key={item.name}
+            item={item}
+            name={item.name}
+            price={item.price}
+            count={item.review_count}
+            image={item.image}
+          />
+        ))}
```

Comment:
```suggestion
        {goods.map(item => {
        const {name, price, review_count, image} = item;
        return(
          <Goods
            key={name}
            item={item}
            name={name}
            price={price}
            count={review_count}
            image={image}
          />
        )})}
```
item을 구조분해할당하라는 의미였습니다!